keanu-project/src/test/java/io/improbable/keanu/algorithms/mcmc/MetropolisHastingsTest.java

```diff
@@ -167,6 +169,40 @@ public void samplesFromPriorWithObservedDeterministic() {
         assertEquals(0.0, postProbTrue, 0.01);
     }
 
+    @Test
+    public void youCanUseAGaussianProposal() {
+
+        DoubleVertex A = new GaussianVertex(20.0, 1.0);
+        DoubleVertex B = new GaussianVertex(20.0, 1.0);
+
+        A.setValue(20.0);
+        B.setValue(20.0);
+
+        DoubleVertex Cobserved = new GaussianVertex(A.plus(B), 1.0);
+
+        Cobserved.observe(46.0);
+
+        BayesianNetwork bayesNet = new BayesianNetwork(Arrays.asList(A, B, Cobserved));
+        bayesNet.probeForNonZeroProbability(100);
+
+        ProposalDistribution proposalDistribution = new GaussianProposalDistribution(DoubleTensor.scalar(1.));
+        MetropolisHastings metropolisHastings = MetropolisHastings.builder()
+            .proposalDistribution(proposalDistribution)
+            .build();
+
+        NetworkSamples posteriorSamples =  metropolisHastings.getPosteriorSamples(
+            bayesNet,
+            Arrays.asList(A, B),
+            1000
+        );
+
+        double averagePosteriorA = posteriorSamples.getDoubleTensorSamples(A).getAverages().scalar();
+        double averagePosteriorB = posteriorSamples.getDoubleTensorSamples(B).getAverages().scalar();
+
+        double actual = averagePosteriorA + averagePosteriorB;
+        assertEquals(44.0, actual, 0.1);
```

Comment:
Can you add a quick comment on how you derived the 44.0 value here please?

Reply:
um... ask @gordoncaleb? I copied it from the test above!

Reply:
If you think about the result of the sum of A and B then you can see that the mu of C is also a gaussian of Gaussian(40, 2). Then if you take the log of their pdfs and sum them you get `(-2(x-40)^2)/8 - ((46-x)^2)/2`. If you find the root of the derivative of that (aka maximum) you will find it's 44. You can see this in WolfgramAlpha `D[(-2 (x - 40)^2)/8 - (46 - x)^2/2, x] == 0`. Maybe put this as a comment on the code?